ext/opcache/zend_accelerator_util_funcs.c

```diff
@@ -425,6 +425,20 @@ static void zend_class_copy_ctor(zend_class_entry **pce)
 			memcpy(trait_aliases[i], ce->trait_aliases[i], sizeof(zend_trait_alias));
 			trait_aliases[i]->trait_method = emalloc(sizeof(zend_trait_method_reference));
 			memcpy(trait_aliases[i]->trait_method, ce->trait_aliases[i]->trait_method, sizeof(zend_trait_method_reference));
+			
+			if (ce->trait_aliases[i]->trait_names) {
+				int j = 0;
+				while (ce->trait_aliases[i]->trait_names[j]) {
+					j++;
+				}
+				trait_aliases[i]->trait_names = emalloc(sizeof(zend_string*) * (j + 1));
+				j = 0;
+				while (ce->trait_aliases[i]->trait_names[j]) {
+					trait_aliases[i]->trait_names[j] = emalloc(sizeof(zend_string));
```

Comment:
no need to alloc/copy here, `trait_aliases[i]->trait_names[j] = ce->trait_aliases[i]->trait_names[j];` should be sufficient here. (I say should; I'm not certain here.)